src/App.js

```diff
@@ -34,6 +81,7 @@ function App() {
 
   const addTodo = (newTodo) => {
     setTodoList([...todoList, newTodo]);
+    postTodo(newTodo);
```

Comment:
the only thing it seems to be missing is to handle the returning value from the POST method to the API. According to the documentation the object the API return looks like this
 ```
{"id":"recu7AWDntmBYNkSI","createdTime":"2023-03-01T11:41:20.000Z","fields":{"title":"make lunch"}}
```

We just need to handle the returning value and adding it to the setTodoList. Right now you are setting the new todo Item before it is added to the database. we just need to add it after we make the POST call. Please let me know if you have any other questions. 

I think the solution would look something like this:
```
const addTodo = (newTodo) => {
   var newTodo =  await postTodo(newTodo);
// handle the value async, response.ok => then add to TodoList
}
```

Reply:
This won;t block you from merging your branch but something to keep in mind.

Reply:
Thank you so much for the feedback @Atos20 
I will try to fix it and will let you know if I need your help :)